props/saber_fett263_buttons.h

```diff
@@ -599,6 +635,10 @@ CUSTOM SOUNDS SUPPORTED (add to font to enable):
 #define FETT263_SAY_BATTERY_PERCENT
 #endif
 
+#if defined(FETT263_EDIT_MODE_MENU) || defined(FETT263_EDIT_SETTINGS_MENU)
+#define FETT263_USE_SETTINGS_MENU
```

Comment:
I recommend renaming this to FETT263__USE_SETTINGS_MENU (note, *two* underscores)
That way you can keep defines which should not be used in config files separate from defines that can/should be used in config files.